airbyte-integrations/connectors/source-outbrain-amplify/source_outbrain_amplify/run.py

```diff
@@ -9,6 +9,7 @@
 from source_outbrain_amplify import SourceOutbrainAmplify
 
 
+
```

Comment:
Dummy change to trigger CI.

Revert when ready;

```suggestion
```

Reply:
Removing this dummy change. See screenshot in previous post for confirmed test output.